ui/imports/shared/popups/RemoveAccountConfirmationPopup.qml

```diff
@@ -1,26 +1,34 @@
 import QtQuick 2.15
 import QtQuick.Controls 2.15
+import QtQml.Models 2.15
 import QtQuick.Layouts 1.15
 
 import StatusQ.Core 0.1
+import StatusQ.Components 0.1
 import StatusQ.Core.Theme 0.1
+import StatusQ.Core.Utils 0.1 as StatusQUtils
 import StatusQ.Controls 0.1
-import StatusQ.Popups 0.1
+import StatusQ.Popups.Dialog 0.1
+
+import AppLayouts.Wallet 1.0
 
 import utils 1.0
 import shared.controls 1.0
 
-StatusModal {
+StatusDialog {
     id: root
 
-    required property bool simple
+    required property string accountType
     required property string accountName
     required property string accountAddress
     required property string accountDerivationPath
+    required property string preferredSharingNetworkShortNames
+    required property string emoji
+    required property string color
 
     signal removeAccount(string address)
 
-    headerSettings.title: qsTr("Remove %1").arg(root.accountName)
+    width: 521
```

Comment:
Should it be a min width then?

Reply:
![image](https://github.com/status-im/status-desktop/assets/491074/eff2cd5e-1285-40ef-bab3-43bf8694a50d)

To answer your question, the account name is being elided in case of a very long one